bip-0390.mediawiki

```diff
@@ -58,6 +58,8 @@ following the <tt>musig()</tt> expression cannot contain
 <tt>/NUMh</tt> or <tt>/NUM'</tt> derivation steps nor <tt>/*h</tt>, or <tt>/*'</tt> child derivation.
 For these <tt>musig()</tt> expressions, the KEY expressions contained within must be xpubs or derived from
 xpubs, and cannot contain child derivation as specified by a <tt>/*</tt>, <tt>/*'</tt>, or <tt>/*h</tt>.
+This restriction on KEY expression child derivation applies even when the derivation steps following the
+<tt>musig()</tt> do not include <tt>/*</tt>.
```

Comment:
I find this hard to understand.

Does this mean that one `musig()` expression may not include two keys where one of the keys is a child derivation of the other key?

Reply:
No, it's saying that `musig(xpub1/*,xpub2/*)/3/4` is disallowed.

I do not see how you could have interpreted it in a way that this was talking about KEY expressions being relative to each other.

Reply:
It might be worth adopting the term "wildcard index" from bip 88? Then you could specify it as "the KEY expression contained within a `musig()` expression cannot include a wildcard index (ie a `/*`, `/*'`, or `/*h`)"? (Or if not that, some other way of talking about derivation templates that generate many keys versus a fixed derivation that always generates a particular key).